### ISSUE TITLE
Chunk Loaders Replacement

### ISSUE THREAD

**Issue by Solidflip** [RTRGitHub/RTRRedux#68]
Currently the only ways to load your base is via the dimensional anchor or mfr chunk loader. Both of these have issues with loading. Everytime i leave my base for even a split second my mfr machines go into idle mode and i have to break them and replace. (gets very annoying with a planter full of 9 different seeds). Not only this but i have to fix cables going across chunk boundries as well. A chunk loader is supposed to fix these issues, however both loaders we currently have do not.

Therefore i put forward that we add ChickenChunks ( a part of ChickenBones) mod to our pack. This mod adds a chunk loader that (imo) beats all others hands down. And Thats all the mod adds.

"But Solid," I hear you say "Players would make a ton, therefore loading half the world!" I hear you, it would be terrible. Therefore i suggest the recipe be disabled, and only allow 1 loader per group (this would be more than enough space for the normal base size). This loader would have to be traded by an admin for the required recipes. Thereby limiting the area loaded. You can also set the amount of chunks each player is allowed to load with these. 

In conclusion i believe this mod will be a great asset to the pack. and now that we are on the ATLauncher adding it in is much easier.

**Comment by CalenMC** [comment]
We used to have ChickenChunks.

But after some weeks some players stop playing and forget to disable their chunk loader, thus leaving a strain on the server without their chunks truly having to be loaded.

That's why we moved to a different way of chunk loading that requires fuel to remain active.

**Comment by CalenMC** [comment]
Actually, not sure if we had ChickenChunks.. possibly. Don't know for sure.

**Comment by levi758336** [comment]
We did have at one point. If all chunk loaders are registered with the admins then it's relatively simple to TP over there and see if shit is overflowing or if a player hasn't been active.

**Comment by solkkku** [comment]
I would cast my vote to chicken chunks as well. All MFR machines stop working when you leave the server or go far away enough from the chunk they are in (even though the chunk should be loaded). The idle bar on the right of the MFR GUI is stuck at  5 / 5t. I've tried this with both the Dimensional anchor and MFR chunk loader. I have also tried to change out the TE itemducts in the farm to a compact AE-system (chest behind harvester and import bus on the chest) to handle the storing crops & moving seeds to the planter. Also tried changing the power source from TE magmatic dynamos & redstone energy conduits to Mekanism Solar panels and universal cables. It does work fine with all methods when I'm in the base myself, but when I leave, the same problem starts again.

Imagine a mob spawner in a chunkloaded base (Dimensional anchor), and mobs keep spawning on cursed earth. Then you disconnect and the MFR grinder stops working (chunk unloads even though it's supposed to be loaded with the current means of chunkloading). Then they will spawn and spawn, and soon it's too much for the server to handle.

Also, all kinds of automated farming with MFR renders itself obsolete with current chunkloaders. Automated farming stops when you leave, and when you come back you have to break and replace all planters and harvesters in your base. This can be a bit tedious to say the least and to collect crops you have to lurk around in your base.

**Comment by Volaunis** [comment]
I've found another possible culprit for at least the MFR idle issues.
Like others, I've had MFR machine go into stuck-idle, but it was only my mobfarm grinders/autospawner, not my farm.
After a bit of testing, i now suspect that the culprit is : the stargate!
I believe the stargate might be doing some kind of weird chunkloading in a 3x3 or 5x5 around it, since that includes my mobfarm, but not the plant farm.
After taking down the gate, I've had no issues with the mobfarm at all, even after logging off, or leaving the area.
This also explains why i didn't have issues with my mobfarm at the start, as i added the gate after i made the mobfarm.
I've only had it removed for a few hours, so time will tell if it really is the culprit, but it sure seems that way to me.

**Comment by Volaunis** [comment]
ok, err, 'close and comment' did that, yeah...sorry bout that :D

**Comment by Solidflip** [comment]
I will test the stargate thing, mine isnt in the same area as my farm but who knows? Will update in about 15 mins

EDIT:Preliminary tests support vols hypothesis. I removed my stargate and my farm seems to be working, Im going to leave the game for about 1 mins to cook dinner and make sure the chunks would have unloaded.

**Comment by Solidflip** [comment]
So today i hit the same issue, The farms stopped working even after removing the stargate. So it appears to be an issue with either MFR or with the dimensional anchors.
